src/src.pro

```diff
@@ -8,7 +8,7 @@ PLUGIN_IMPORT_PATH = io/thp/pyotherside
 
 TEMPLATE = lib
 CONFIG += qt plugin
-QT += qml quick
+QT += qml quick svg
```

Comment:
Yeah, I also wondered about this. There can definitely be cases when this is not needed and the the added dependencies could definitely add overhead.

But I think you will nee to do this as I'm really not that versed in the Qt project configuration files to know how to do it. :)